### ISSUE TITLE
Mining zerodot1 - domain "be" vs "www.be"

### ISSUE THREAD

**Issue by driesg** [naveednajam/Little-Snitch---Rule-GroupsV2#4]
Hey,

thanks for this tool - it's very helpful.

I've noticed that when I import "mining zerodot1", all traffic to any ".be" website will be blocked. This happens because of the rule `Deny outgoing connections to domain be`. This matches any website with a top level of .be of course. I've checked the dotzero1 list and the entry there is actually `www.be`, which is a valid full domain rather than just `be`. 

I assume this may be a bug in the script that generates the rules? Would that be possible?

**Comment by driesg** [comment]
This could be the cause:

https://github.com/naveednajam/Little-Snitch---Rule-GroupsV2/blob/39d3b9981e/updateSources.py#L412-L417

A solution might be to _not_ remove `www` if only the TLD remains

**Comment by naveednajam** [comment]
@driesg  Thanks for high lighting this issue, the issue is caused by the UPPER case TLDS in tlds list, as to avoid similar issue I filter out all the valid TLDS from rules. this has happened before however previously I was unable to find the root cause. Only now I noticed that TLDS in the list are sometime in upper case letter which failed the match and so failed to remove the tlds from the rules. I have fixed the issue. updates your list and let me know if this have been solved.
